test/Microsoft.Azure.Devices.Edge.Test/EdgeAgentDirectMethods.cs

```diff
@@ -57,8 +56,8 @@ public async Task TestGetModuleLogs()
                 Context.Current.NestedEdge);
             await Task.Delay(30000);
 
-            string since = DateTime.Now.AddDays(-1).ToString("yyyy-MM-dd'T'HH:mm:sZ");
-            string until = DateTime.Now.AddDays(+1).ToString("yyyy-MM-dd'T'HH:mm:sZ");
+            string since = DateTime.Now.AddDays(-1).ToString("yyyy-MM-dd'T'HH:mm:ssZ");
+            string until = DateTime.Now.AddDays(+1).ToString("yyyy-MM-dd'T'HH:mm:ssZ");
```

Comment:
Trying to udnerstand the datetime format difference. So the old format only had tens of a second displaying?

Reply:
Yes, in case the seconds field is less than 2 digits